lib/core/middleware/nuxt.js

```diff
@@ -70,7 +70,7 @@ export default async function nuxtMiddleware(req, res, next) {
 
     if (this.options.render.csp) {
       const { allowedSources, policies } = this.options.render.csp
-      let cspStr = `script-src 'self'${this.options.dev ? " 'unsafe-eval'" : ''} ${(cspScriptSrcHashes).join(' ')}`
+      let cspStr = `script-src 'self'${this.options.dev ? ` 'unsafe-eval'` : ''} ${(cspScriptSrcHashes).join(' ')}`
```

Comment:
Is this change necessary ?

Reply:
Not necessary, did this because my editor settings replace `"'...'"` with `'\'...\''`. Should I revert it?

Reply:
I think we can:

```js
let cspStr = `script-src 'self'${this.options.dev ? ' unsafe-eval' : ''} ${(cspScriptSrcHashes).join(' ')}`
```

Reply:
The "double quotes" are necessary.

The final must should look like: `script-src 'self' 'unsafe-eval' ...`

Reply:
Sorry, my mistake, I didn't notice that.